src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -401,8 +401,11 @@ protected function cleanup($dataType, $data)
 
         // Delete Files
         foreach ($dataType->deleteRows->where('type', 'file') as $row) {
-            foreach (json_decode($data->{$row->field}) as $file) {
-                $this->deleteFileIfExists($file->download_link);
+            $field = json_decode($data->{$row->field});
+            if (is_array($field)) {
```

Comment:
You can simplify this a bit with something like:
```
if (is_array($field = json_decode($data->{$row->field}))) {
```

But beyond that, is there any reason passing "true" into `json_decode` as the 2nd argument wouldn't work?

Reply:
Yes thanks, it is better.

Passing true to json_decode still returns null and raises the same error.